vg/draw/canvas.go

```diff
@@ -276,24 +314,59 @@ func (c *Canvas) Y(y float64) vg.Length {
 	return vg.Length(y)*(c.Max.Y-c.Min.Y) + c.Min.Y
 }
 
-// Crop returns a new Canvas corresponding to the receiver
-// area with the given number of inches added to the minimum
+// Crop returns a new Canvas corresponding to the Canvas
+// c with the given lengths added to the minimum
 // and maximum x and y values of the Canvas's Rectangle.
-func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
+// Note that cropping the right and top sides of the canvas
+// requires specifying negative values of right and top.
+func Crop(c Canvas, right, bottom, left, top vg.Length) Canvas {
 	minpt := Point{
-		X: c.Min.X + minx,
-		Y: c.Min.Y + miny,
+		X: c.Min.X + right,
+		Y: c.Min.Y + bottom,
 	}
 	maxpt := Point{
-		X: c.Max.X + maxx,
-		Y: c.Max.Y + maxy,
+		X: c.Max.X + left,
+		Y: c.Max.Y + top,
 	}
 	return Canvas{
-		Canvas:    vg.Canvas(c),
+		Canvas:    c,
 		Rectangle: Rectangle{Min: minpt, Max: maxpt},
 	}
 }
 
```

Comment:
Do you mean the "Tiles" type? If so, I added one, but suggestions on the wording are welcome.